test/data/vcf2maf/Makefile

```diff
@@ -0,0 +1,28 @@
+SHELL=/bin/bash
+VCF2MAF_VERSION=v1.6.15
+VCF2MAF_URL=https://raw.githubusercontent.com/mskcc/vcf2maf/$(VCF2MAF_VERSION)/
```

Comment:
you forgot to use this one

Reply:
fixed. Also pointed it to use cyriac's updates to the test cases that includes the `G - A` issue